BetterStats/BetterStats.cs

```diff
@@ -665,7 +674,7 @@ public static void AddPlanetFactoryData(PlanetFactory planetFactory)
                 if (assembler.id != i || assembler.recipeId == 0) continue;
 
                 var frequency = 60f / (float)((double)assembler.timeSpend / 600000.0);
-                var speed = (float)(0.0001 * (double)assembler.speed);
+                var speed = (float)(0.0001 * Math.Max(assembler.speedOverride, assembler.speed));
```

Comment:
This part accounts for assemblers that have the "Production Speedup" toggle turned on